src/battle_interface.c

```diff
@@ -915,13 +944,13 @@ static void PrintSafariMonInfo(u8 healthboxSpriteId, struct Pokemon *mon)
         u8 elementId;
 
         if ((text[j] >= 55 && text[j] <= 74) || (text[j] >= 135 && text[j] <= 154))
-            elementId = HEALTHBOX_GFX_44;
+            elementId = B_INTERFACE_GFX_SAFARI_HEALTHBOX_1;
         else if ((text[j] >= 75 && text[j] <= 79) || (text[j] >= 155 && text[j] <= 159))
```

Comment:
This more of a note to self, but I wonder if it'd make sense at some point to add JP charmap constants, given they're still used in places like this.